File: cs50/week6/pset6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


while True:
    n = get_int("Size: ")
    if (n > 0) and (n < 9):
        break

for i in range(n):

    j = n - (i+1)
    while j > 0:
        print(" ", end="")
        j-=1

    l = n - (i+1)

    while l < n:
        print("#", end="")
        l+=1

    print("  ", end="")

    t = n - (i+1)

    while t < n:
        print("#", end="")
        t+=1


    print("")

File: cs50/week1/pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
// gets height from user
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while(n < 1);

    
    // main loop
    for(int i = 0; i<n;i++){

        // print white spaces
        for(int j = n-(i+1); j > 0;j--)
        {
            printf(" ");
        }

        //prints first column of #
        for(int l = n-(i+1); l < n;l++)
        {
            printf("#");
        }

        // prints gap
        printf("  ");

        // prints 2nd column
        for(int t = n-(i+1); t<n;t++)
        {
            printf("#");
        }

        //newline
        printf("\n");
    }
}